src/REL/d/a/obj/d_a_obj_tarzan_pole.cpp

```diff
@@ -1,3 +1,111 @@
 #include "d/a/obj/d_a_obj_tarzan_pole.h"
 
+#include "d/a/d_a_base.h"
+#include "d/a/d_a_item.h"
+#include "d/a/d_a_itembase.h"
+#include "d/a/d_a_player.h"
+#include "d/col/bg/d_bg_s.h"
+#include "d/col/cc/d_cc_d.h"
+#include "d/col/cc/d_cc_s.h"
+#include "f/f_base.h"
+#include "nw4r/g3d/res/g3d_resmdl.h"
+#include "toBeSorted/attention.h"
+
 SPECIAL_ACTOR_PROFILE(OBJ_TARZAN_POLE, dAcOTarzanPole_c, fProfile::OBJ_TARZAN_POLE, 0x235, 0, 3);
+
+// clang-format off
+dCcD_SrcSph dAcOTarzanPole_c::sCcSrc = {
+    /* mObjInf */ {
+        /* mObjAt */ {0, 0, {0, 0, 0}, 0, 0, 0, 0, 0, 0},
+        /* mObjTg */ {0xA00, 0x800311, {0, 0, 0x407}, 0x0000, 0x0000},
```

Comment:
```suggestion
        /* mObjTg */ {(AT_TYPE_WHIP | AT_TYPE_WIND), 0x800311, {0, 0, 0x407}, 0x0000, 0x0000},
```
for At and Tg we know the type fields. reference the `dCcD_ObjAtType` enum.